Jenkinsfile

```diff
@@ -65,18 +67,107 @@ pipeline {
                 }
             }
         }
+        stage('Assemble Jupyter Notebook Docker files') {
+            when {
+                environment name: 'SKIP_BUILD', value: 'false'
+            }
+            // agent {
+            //     docker {
+            //         image 'python:3.7'
+            //         args '--network=host'
+            //         reuseNode true
+            //     }
+            // }
+            agent {
+                docker {
+                    image 'ktaletsk/polus-railyard:0.3.1'
```

Comment:
I can create a labshare Docker Hub repo for this.

Reply:
Fixed